tmpfiles.d/meson.build

```diff
@@ -2,17 +2,18 @@
 
 enable_tmpfiles = conf.get('ENABLE_TMPFILES') == 1
 
-files = [['README',               ''],
-         ['home.conf',            ''],
-         ['journal-nocow.conf',   ''],
-         ['systemd-nologin.conf', 'HAVE_PAM'],
-         ['systemd-nspawn.conf',  'ENABLE_MACHINED'],
-         ['systemd-resolve.conf', 'ENABLE_RESOLVE'],
-         ['systemd-tmp.conf',     ''],
-         ['portables.conf',       'ENABLE_PORTABLED'],
-         ['systemd-pstore.conf',  'ENABLE_PSTORE'],
-         ['tmp.conf',             ''],
-         ['x11.conf',             ''],
+files = [['README',                ''],
+         ['home.conf',             ''],
+         ['journal-nocow.conf',    ''],
+         ['systemd-nologin.conf',  'HAVE_PAM'],
+         ['systemd-nspawn.conf',   'ENABLE_MACHINED'],
+         ['systemd-resolve.conf',  'ENABLE_RESOLVE'],
+         ['systemd-tmp.conf',      ''],
+         ['portables.conf',        'ENABLE_PORTABLED'],
+         ['systemd-pstore.conf',   'ENABLE_PSTORE'],
+         ['tmp.conf',              ''],
+         ['x11.conf',              ''],
+         ['systemd-networkd.conf', 'ENABLE_NETWORKD'],
```

Comment:
For resolved, we have systemd-resolve.conf. So, maybe systemd-network.conf is better??

Note, we have systemd-network.conf for sysusers.